app/src/main/java/mod/hilal/saif/activities/tools/AppSettings.java

```diff
@@ -146,6 +148,12 @@ private void createToolsView(int icon, String title, String desc, LinearLayout t
         LibraryItemView item = new LibraryItemView(this);
         item.enabled.setVisibility(View.GONE);
         item.icon.setImageResource(icon);
+
+        TypedValue typedValue = new TypedValue();
+        getTheme().resolveAttribute(android.R.attr.colorPrimary, typedValue, true);
+        int colorPrimary = typedValue.data;
```

Comment:
What about to create another method in `ThemeUtils` to get theme color.

Maybe `ThemeUtils#getColorFromTheme`?

Reply:
MaterialColors#getColor, doesn't it already do that?